lib/genParser.ml

```diff
@@ -237,6 +239,7 @@ module Make
               MiscParser.condition =
               ConstrGen.set_kind k parsed.MiscParser.condition; } in
       let parsed =
+        if not O.set_hash then parsed else
```

Comment:
Wouldn't be more simple to intervene at the printing step? This may involve less changes of configuration modules.